mysql-test/suite/galera_sr/r/galera_sr_shutdown_master.result

```diff
@@ -1,5 +1,7 @@
 connection node_2;
 connection node_1;
+connection node_1;
+connection node_2;
```

Comment:
Are you sure this is required ? , For me test case passes without this .
worker[1] Using MTR_BUILD_THREAD 300, with reserved ports 16000..16019
galera_sr.galera_sr_shutdown_master 'innodb' [ pass ]  40697
galera_sr.galera_sr_shutdown_master 'innodb' [ 2 pass ]  30692
galera_sr.galera_sr_shutdown_master 'innodb' [ 3 pass ]  40630